### PR TITLE
ic.cz: ignores

### PR DESCRIPTION
**Pull request by Sanqui**

### CODE REVIEW

File: db/ignore_patterns/internetcentrum.json [hunk 1]
```diff
@@ -56,6 +63,7 @@
     "profile\\.php\\?mode=email",
     "profile\\.php\\?mode=register",
     "\\?q=event.+/(?:day|list|month|table|week)/all/all",
+    "random_num=\d+",
```
Comment (ivan) [inline db/ignore_patterns/internetcentrum.json]: you need `\\d`

File: db/ignore_patterns/internetcentrum.json [hunk 2]
```diff
@@ -1,6 +1,9 @@
 {
   "name": "internetcentrum",
   "patterns": [
+    ".*\..*\..*\.pl",
```
Comment (ivan) [inline db/ignore_patterns/internetcentrum.json]: needs `\\.`

**Comment by Sanqui** [comment]
Duh, I didn't realize it's json.  Sorry.

File: db/ignore_patterns/internetcentrum.json [hunk 3]
```diff
@@ -43,8 +49,10 @@
     "modules\\.php\\?name=coppermine.*meta=(?:topn|toprated|lastcom|lastup|lastupby|random|lastcomby)",
     "modules\\.php\\?name=Statistics",
     "name=Kalender",
+    "name=Statistics",
```
Comment (yipdw) [inline db/ignore_patterns/internetcentrum.json]: Are there URLs that have just `name=Statistics`, aren't `modules.php?name=Statistics`, but produce similar content?

(I've not seen them but that definitely doesn't mean anything)

Comment (Sanqui) [inline db/ignore_patterns/internetcentrum.json]: I believe so, I've seen some.  They're similar sinks as calendars.

**Pull request merged**